crates/ide/src/completion/complete_postfix/format_like.rs

```diff
@@ -0,0 +1,325 @@
+//! Postfix completion for `format`-like strings.
```

Comment:
Let's add a `Feature:` header here and turn this into docs, a-la

* https://github.com/rust-analyzer/rust-analyzer/blob/master/crates/ide/src/completion.rs#L41
* https://rust-analyzer.github.io/manual.html#magic-completions

(I think you need to change `//!` to `//` for this to work)